docs/index.md

```diff
@@ -115,12 +115,17 @@ Travis will then kick off a build against the new tag created and deploy that bu
 
 There are a few things that can be checked if you review a pull request against one of our modules:
 
-* Correct email address used in the commits (if so, travis displays the avatar next to the commit)?
+* Does the email address used in the commits match the github email address? (This will let github display the contributor's avatar next to the commit)
 * Is this a bugfix, modulesync, breaking change, enhancement, docs update? Label it with `bug`, `modulesync`, `backwards-incompatible`, `enhancement`, `docs`
 * Are updates to the README.md needed but missing? Label it with `needs-docs`
+* Has the file documented params or examples in the header? This needs to be updated as well
 * Are there merge conflicts? Add the `needs-rebase` label
 * Does it need additional tests? Add the `needs-tests` label
-* Are new parameters introduced? They should have datatypes
+* Does it have failing tests? Add the `tests-fail` label
+* Are new parameters introduced? They must have datatypes
 * Are facts used? They should only be accessed via `$facts[]` or [fact()](https://github.com/puppetlabs/puppetlabs-stdlib#fact) from stdlib, but not topscope variables
+* Varibales shouldn't be accessed via topscope: $::modulename::$param. Instead do: $modulename::$param
```

Comment:
spelling of 'variables'.  Anyway, is this true?  I thought it was classes that definitely didn't need to be accessed by topscope anymore.  Variables, not sure.

Reply:
Should it be ```$::modulename::param``` rather than ```$param```?

Reply:
What I wanted to say is: You shouldn't use $:: anymore, no matter if it's a class, param or fact.

Reply:
updated it

Reply:
Are you sure?  I thought it was just classes where this was 100% true.  and what about parameters from the ENC?

Reply:
Aren't parameters from the ENC presented as facts? Haven't checked this tbh but I see them in the same way.

Reply:
There is no dynamic scoping of variables or class names anymore.